src/app/components/password.component.ts

```diff
@@ -0,0 +1,32 @@
+import { PasswordBadge, DebounceHelper, InputValidator, PasswordToggleButton } from '../components';
+
+export class PasswordComponent{
+    public static create(initialState: string) : HTMLDivElement{
+        const passwordDiv = document.createElement('div');
+
+        const passwordLabel = document.createElement('label');
+        Object.assign(passwordLabel, {innerText : 'Password', htmlFor: 'passwordInput'});
+
+        const passwordInput = document.createElement('input');
+        Object.assign(passwordInput, {id: 'passwordInput', type: 'password', minLength: 8, maxLength: 60});
+
+        const badge = PasswordBadge.create(initialState);
+        const debounce = new DebounceHelper();
+        const validator = new InputValidator(); 
```

Comment:
Please, check do we really need to create an instance for this validator ?

Reply:
not need